### PR TITLE
update base css bundle content

### PR DESCRIPTION
**Pull request by wen-2018**
## One-line summary

This PR:

- add new nav and footer styles to the new base CSS bundle
- remove old nav and footer styles from the new base CSS bundle
- remove redundant styles from templates
- uses the new proposed one switch for website refresh work(issue [#15289](https://github.com/mozilla/bedrock/issues/15289))

## Issue / Bugzilla link

https://github.com/mozilla/bedrock/issues/15120

## Testing
When switch `m24-website-refresh` is on, the new nav and footer should look and work as before.

### CODE REVIEW

**Review by craigcook — CHANGES_REQUESTED** [review]

File: bedrock/base/templates/base-protocol-mozilla.html [hunk 1]
```diff
@@ -9,11 +9,8 @@
 {% block body_class %}mzp-t-mozilla{% endblock %}
 
 {% block site_css %}
-  {% if switch('m24-website-refresh') %}
+  {% if switch('m24-website-refresh'), ['en'] %}
```
[the remark this answers is not in the document]
Comment (wen-2018) [reply bedrock/base/templates/base-protocol-mozilla.html]: I want to make sure that I am understanding the new single switch for M24 correctly. 

This is how I understand it:  If the `m24-website-refresh` switch is on and the content is available in the locales we support (currently just English), we will display the new designs (new nav, new footer, new home, new about etc) + new fonts for old designs for the users. If the language the user chooses is not english, we will display none of the new designs, nor the new fonts or new logos for the existing designs.

I think the code block here reflects this understanding. 
```
  {% if switch('m24-website-refresh', ['en']) %}
    {{ css_bundle('protocol-mozilla-2024') }}
  {% else %}
    {{ css_bundle('protocol-mozilla') }}
  {% endif %}
  ```
Or is the intention for one switch for the refresh work is something else?

[the remark this answers is not in the document]
Comment (craigcook) [reply bedrock/base/templates/base-protocol-mozilla.html]: The global styles (fonts etc) that are defined in the `protocol-mozilla-2024` bundle should be for all languages.

The only locale-specific portions of this refresh are the home and about pages, and the nav and footer. So we can load the nav and footer styles for all languages but anything other than English won't display those components so the styling isn't needed for those languages. Non-English pages will still need the old nav and footer styles. 

So I think we need to create separate CSS bundles for new nav-and-footer and old nav-and-footer, then load those two bundles with locale-specific switch logic:
```
  {% if switch('m24-website-refresh', ['en']) %}
    {{ css_bundle('m24-nav-and-footer') }}
  {% else %}
    {{ css_bundle('legacy-nav-and-footer') }}
  {% endif %}
```

[the remark this answers is not in the document]
Comment (wen-2018) [reply bedrock/base/templates/base-protocol-mozilla.html]: I see what you mean now. I think I might have a solution without loading redundant CSS

**Comment by craigcook** [comment]
This seems to work but I'm not sure it needs to be this complicated. I could be wrong, but I still don't think we need the extra level of importing in Sass and making two different Protocol packages.

I think there can be a single bundle that includes all the M24 general styles (minus nav and footer components) that gets loaded on every page if the switch is on. Then two separate bundles for ONLY the navs and footers, and use language logic to determine which bundle to load. And if the switch is off, just fall back to the old Protocol (which already includes the old nav and footer). 

```
     {% if switch('m24-website-refresh') %}
        {{ css_bundle('protocol-mozilla-2024') }}
        {% if LANG.startswith('en-') %}
          {{ css_bundle('m24-navigation-and-footer') }}
        {% else %}
          {{ css_bundle('legacy-navigation-and-footer') }}
        {% endif %}
      {% else %}
        {{ css_bundle('protocol-mozilla') }}
      {% endif %}
```

It does mean loading two bundles per page, but almost every page already loads at least two bundles already, and many load four or five. I don't see the benefit in further complicating the source just to avoid loading an extra style sheet in the client. We've been using HTTP/2 for some time so the additional requests aren't the performance hit they used to be.

I haven't tested it but I think this can solve the problem and would keep the source tidier, but I could be overlooking something. I'm happy to discuss further.

**Pull request closed without merging**